end_to_end/music_recommendation/03_train_deploy_debugger_explain_monitor_registry.ipynb

```diff
@@ -4,48 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        #     base_job_name=model_name,
</pre><p>Remove commented line? Would it be good to provide a base job name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Removed - it is manually created using datetime and provided instead in the fit function. I needed to exact model name for subsequent steps. Using base_job_name here would make retrieve the exact model name a little more inconvienient</p>